CMakeLists.txt

```diff
@@ -70,9 +70,32 @@ option(REDUCE_EXPORTS "Attempt to reduce exported symbols in the resulting execu
 option(WERROR "Treat compiler warnings as errors." OFF)
 
 tristate_option(CCACHE "Use ccache for compiling." "if ccache is found." AUTO)
-tristate_option(WITH_NATPMP "Enable NAT-PMP." "if libnatpmp is found." AUTO)
-tristate_option(WITH_MINIUPNPC "Enable UPnP." "if libminiupnpc is found." AUTO)
-tristate_option(WITH_ZMQ "Enable ZMQ notifications." "if libzmq is found." AUTO)
+
+option(WITH_NATPMP "Enable NAT-PMP." OFF)
```

Comment:
Aren't we meant to be reming these, or is that happening at a later point?

Reply:
Yes, I'm going to rename all options at once at a later point.